Server/MirObjects/PlayerObject.cs

```diff
@@ -3971,7 +3984,7 @@ public void Chat(string message, List<ChatItem> linkedItems = null)
                             ReceiveChat(string.Format("You have deleted skill {0} from player {1}", skill1.ToString(), player.Name), ChatType.Hint);
                             player.ReceiveChat(string.Format("{0} has been removed from you.", skill1), ChatType.Hint);
 
-                            Helpers.ChatSystem.SystemMessage(chatMessage: $"{player} Skill {skill1.ToString()} was removed by GM: {Name}");
+                            Helpers.ChatSystem.SystemMessage(chatMessage: $"{player.Name} Skill {skill1.ToString()} was removed by GM: {Name}");
```

Comment:
does "chatMessage:" need to be removed here like the above one?

Reply:
Once you @DeleteSkill it would display Server.MirObjects.PlayerObjects Skill FireWall was removed by GM: X

@KILL didn't display anything. It has the kill message but never work, So whether we want this? or remove.